zcash_client_sqlite/src/wallet/transparent.rs

```diff
@@ -1372,9 +1363,9 @@ pub(crate) fn put_transparent_output<P: consensus::Parameters>(
              mined_height = :mined_height
          RETURNING id_tx",
         named_params![
-           ":txid": &outpoint.hash().to_vec(),
+           ":txid": &output.outpoint().hash().to_vec(),
            ":block": block,
-           ":mined_height": output_height
+           ":mined_height": output.mined_height().map(u32::from)
```

Comment:
The existing code was correct:
```suggestion
           ":mined_height": output_height
```